opencti-platform/opencti-front/src/private/components/common/files/import_files/ImportFilesContext.tsx

```diff
@@ -190,8 +190,11 @@ export const ImportFilesProvider = ({ children, initialValue }: {
   const canSelectImportMode = useGranted(['KNOWLEDGE_KNASKIMPORT']); // Check capability to set connectors and validation mode
   const draftContext = useDraftContext();
 
-  const [activeStep, setActiveStep] = useState(initialValue.activeStep ?? (canSelectImportMode ? 0 : 1));
-  const [importMode, setImportMode] = useState<ImportMode | undefined>(!canSelectImportMode ? 'auto' : initialValue.importMode);
+  const initalActiveStep = initialValue.activeStep ?? (canSelectImportMode ? 0 : 1);
+  const initialImportMode = !canSelectImportMode ? 'auto' : initialValue.importMode;
```

Comment:
prefer positive way :)

```suggestion
  const initialImportMode = canSelectImportMode ? initialValue.importMode : 'auto';
```

Reply:
indeed that's better :)